app/controllers/users_controller.rb

```diff
@@ -24,7 +24,9 @@ def create
         rescue
           flash[:warning] = "We tried and failed to send you a welcome email, but your account was created anyhow. Sorry!"
         end
-        flash[:notice] = I18n.t('users_controller.registration_successful').html_safe
+        flash[:notice] = I18n.t('users_controller.registration_successful')
+        flash[:notice] += " " + I18n.t('users_controller.continue_where_you_left_off', :url1 => params[:return_to].to_s) if params[:return_to] && params[:return_to] != "/signup"
```

Comment:
Can you please explain your changes here?

Reply:
This adds the link to return to the page the user signed in from.

Reply:
![image](https://user-images.githubusercontent.com/44309027/50540529-0d9e8c00-0b59-11e9-9dad-e708625dfbc6.png)

Reply:
Awesome!!!
From next time please create separate issues and separate solutions to avoid confusion. 
Sorry for misunderstanding.
I loved it.